### PR TITLE
Created a welcomebot  for this pi-builder project

### PR DESCRIPTION
**Pull request by prius2055**
Created a config,yml file, a welcome bolt for this project

Thanks for opening a pull request! In this repository, opening a PR will initiate the generation of a new Raspberry Pi image, and create an image file you can download and use in your Raspberry Pi.

The changes you add to the pull request, such as adding software to install, will be run on the generated image.

For an example, see the software installed and configured in this pull request: https://github.com/publiclab/image-builder-rpi/pull/15/files

### Recipe

Use this space to describe what your "recipe" is intended to install and configure on a Raspberry Pi:

****

### Download instructions

Generating the image will take a few minutes. Once the image is prepared, and if it succeeded, you'll see a green checkmark at the bottom of the pull request. To download the image:

1. click the green checkmark; you'll go to a page at a URL like `https://gitlab.com/publiclab/image-builder-rpi/pipelines/########/builds`
2. On this page, click the `Jobs` tab, next to `Pipeline`
3. Click the green `Passed` button
4. Click `Download` in the right-hand sidebar
5. Unzip the `artifacts.zip` file, and also the `hypriotos-rpi-camera_web.img.zip` within it
6. Use a program like https://etcher.io/ to flash it to an SD card

You'll also be able to read the output of the image generation in this window.

We hope to create a bot to report back the completed image URL in each pull request. If you can help create such a bot, please contact us at:

https://github.com/publiclab/image-builder-rpi/issues/16

Thanks!

### CODE REVIEW

**Comment by prius2055** [comment]
@jywarren Kindly review my PR

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
+ 
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
+ Do join our weekly check-in to share your this week goal and the awesome work you did 😃. Please find the link **pinned in the issue section** 📝
```
Comment (cesswairimu) [inline .github/config.yml]: you can remove this part as we are not running weekly check-ins

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ Also please refer [here](https://github.com/publiclab/plots2/issues/3840) for installation help 💿
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
+ 
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
```
Comment (cesswairimu) [inline .github/config.yml]: This part is specifically for the plots2 project, you could  remove this or mention automatic builds here

**Review by cesswairimu — CHANGES_REQUESTED** [review]
Thanks for opening this @prius2055, there are a few things that do not relate to this project here, could you please go through the text and remove/edit them,,,, the readme is a good place to find most of the info https://github.com/publiclab/pi-builder#readme, but I had highlighted a few. Thanks

File: .github/config.yml (hunk 2 above)
Comment (prius2055) [reply .github/config.yml]: I have removed the unwanted line

**Review by prius2055 — COMMENTED** [review]
I have removed the unwanted line. Please review

File: .github/config.yml (hunk 1 above)
Comment (prius2055) [reply .github/config.yml]: I have removed the unwanted line

**Comment by prius2055** [comment]
Thank you @cesswairimu, I have removed the unwanted line. Please review

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  Thanks for opening your first issue! This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+  Please follow the issue template to help us help you 👍🎉😄
+  If you have screenshots [or a gif](https://publiclab.org/gifs/) to share demonstrating the issue, that's really helpful! 📸
+  Do join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
```
Comment (cesswairimu) [inline .github/config.yml]: remove line 16 and 17

File: .github/config.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ Also please refer [here](https://github.com/publiclab/plots2/issues/3840) for installation help 💿
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
+ 
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
```
Comment (cesswairimu) [inline .github/config.yml]: This line I had mentioned earlier https://github.com/publiclab/pi-builder/pull/79/files#r1003170965

**Review by prius2055 — COMMENTED** [review]
I have made changes and effected the corrections as indicated

**Comment by prius2055** [comment]
Thank you @cesswairimu for your patience with me. I have made the necessary adjustment. Kindly review

**Comment by prius2055** [comment]
Hi @cesswairimu I'm yet to get your review. Kindly review

File: .github/config.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
+ It would be great if you can tell us your Twitter handle so we can thank you properly?
+ 
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
```
Comment (cesswairimu) [inline .github/config.yml]: hey, you missed this line again! This is specific for plots2 project, as code from here is not published on stable.publiclab.org and publiclab.org...does that make sense?

Comment (prius2055) [reply .github/config.yml]: Hi @cesswairimu the correction has been effected. Sorry for the errors

Comment (prius2055) [reply .github/config.yml]: The unnecessary lines have been removed

**Comment by prius2055** [comment]
@cesswairimu please review again, I have made the correction. Thanks for your patience

**Review by cesswairimu — APPROVED** [review]
Looks good, thanks

**Comment by cesswairimu** [comment]
@jywarren please help merge this when you have a sec. thanks

**Pull request closed without merging**